lib/src/actions/user/providers/mod.rs

```diff
@@ -6,6 +6,8 @@ use self::none::NoneUserProvider;
 use super::UserVariant;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+mod linux;
+use self::linux::LinuxUserProvider;
 
 #[derive(JsonSchema, Clone, Debug, Serialize, Deserialize)]
 pub enum UserProviders {
```

Comment:
A minor nit. You could get rid of `...UserProvider`. I think, that's why clippy is complaining.

But you can merge if you solve it later :+1: